docs/index.mdx

```diff
@@ -12,7 +12,19 @@ import CustomDocCardList from '@site/src/components/CustomDocCardList'
 <div className="grid-container">
   <div>
 
-DevCycle is a feature flag platform built for teams of any size, helping you easily create, rollout, and cleanup feature flags without disrupting your workflow. With integrations for tools you are already using like [GitHub](integrations/github/feature-usage-action), [Datadog](integrations/datadog-rum), and [Jira](integrations/jira), DevCycle is ready to add to your project today!
+DevCycle is a feature flag platform built for teams of any size, helping you easily create, rollout, and cleanup feature flags without disrupting your workflow.
+
+    We support everything you expect from a feature management platform:
+    - **SDKs for Every Language**
+    - **Fast, Global APIs**
+    - **Powerful and Reusable Targeting Rules**
+    - **A/B Testing and Experimentation**
+    - **Scheduled Releases and Rollouts**
+    - **Realtime Updates to All Users**
+    - **Permissions and Audit Logs**
+    - **Integrations with Your Favorite Tools**
+
+    With tons of built-in features, DevCycle is ready to add to your project today!
 
 **[Follow our Quickstart](/introduction/quickstart) to learn how everything works, or [install an SDK](sdk) for your favorite language to start adding feature flags to a new or existing project.**
```

Comment:
I would delete this text, it's redundant to the blocks below

Reply:
sort of disagree, it's a pretty good "what to do next" statement thats a bit clearer than the list of boxes of sections that appear below it